pages/playground.js

```diff
@@ -7,15 +7,28 @@ import Button from "@components/Button";
 import PreviewModal from "@components/modals/ProfilePreview";
 import Input from "@components/form/input";
 
+  const defaultJson = `{
+  "name": "Your Name",

```

Comment:
Formatting looks a bit off, please use the VScode Prettier plugin

Reply:
Sure, Thanks for reviewing it